### PR TITLE
Implemented Scarwood Bandits

### PR DESCRIPTION
**Pull request by Zzooouhh**
Caution: I've been unable to test this card in my current environment.

### CODE REVIEW

File: Mage.Sets/src/mage/cards/s/ScarwoodBandits.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        if (controller != null) {
+            Permanent targetArtifact = game.getPermanent(getTargetPointer().getFirst(game, source));
+            if (targetArtifact != null) {
+                Player player = game.getPlayer(targetArtifact.getControllerId());
+                if (player != null) {
+                    cost.clearPaid();
+                    if (player.chooseUse(Outcome.Benefit, "Pay {2}? (Otherwise Scarwood Bandits' controller gains control of" + targetArtifact.getLogName() + ")", source, game)) {
+                        cost.pay(source, game, targetArtifact.getControllerId(), targetArtifact.getControllerId(), false, null);
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/s/ScarwoodBandits.java]: This should actually ask each opponent if they want to pay {2} and allow them to pay even if someone else already has.

File: Mage.Sets/src/mage/cards/s/ScarwoodBandits.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+/**
+ *
+ * @author L_J
+ */
+public class ScarwoodBandits extends CardImpl {
+
+    public ScarwoodBandits(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.CREATURE},"{2}{G}{G}");
+        this.subtype.add("Human");
+        this.subtype.add("Rogue");
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/s/ScarwoodBandits.java]: The string subtype is deprecated, use SubType.HUMAN and SubType.ROGUE instead.

**Comment by Zzooouhh** [comment]
Alright, came up with a solution

File: Mage.Sets/src/mage/cards/s/ScarwoodBandits.java [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+            for (UUID playerId : game.getState().getPlayersInRange(controller.getId(), game)) {
+                Player player = game.getPlayer(playerId);
+                if (player != null && player != controller && cost.canPay(source, source.getSourceId(), player.getId(), game) && player.chooseUse(Outcome.Detriment, message, source, game)) {
+                    cost.clearPaid();
+                    if (cost.pay(source, game, source.getSourceId(), player.getId(), false, null)) {
+                        if (!game.isSimulation()) {
+                            game.informPlayers(player.getLogName() + " pays the cost to prevent the effect");
+                        }
+                        doEffect = false;
+                        break;
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/s/ScarwoodBandits.java]: No break needed here, people still get the option to pay even if nobody paid. I know, it's weird.

**Comment by Zzooouhh** [comment]
The original DoUnlessAnyPlayerPaysEffect also includes the break, perhaps I should remove it there as well?

**Comment by theelk801** [comment]
I got it, I'm about to push a commit anyway.

**Pull request merged**